src/index.html

```diff
@@ -4,10 +4,305 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header" id="home">
+      <div class="header__crown">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="crown"
+            class="header__crownLogo"
+          >
+        </a>
+      </div>
+      <div class="header__container">
+        <div class="header__nav">
+          <div class="header__toggler"></div>
+          <nav class="header__nav nav">
+
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#" class="nav__link">home</a>
+              </li>
+              <li class="nav__item">
+                <a href="#about" class="nav__link">about us</a>
+              </li>
+              <li class="nav__item">
+                <a href="#gallery" class="nav__link">gallery</a>
+              </li>
+              <li class="nav__item">
+                <a href="#consultation" class="nav__link">consultation</a>
+              </li>
+              <li class="nav__item">
+                <a href="#testimonials" class="nav__link">testimonials</a>
+              </li>
+              <li class="nav__item">
+                <a href="#contacts" class="nav__link">contacts</a>
+              </li>
+              <li class="nav__contacts">+1 234 555-55-55</li>
+              <li class="nav__contacts">
+                <a
+                  href="tel:+1 234 555-55-55"
+                  class="nav__callLink"
+                >
+                  call to order
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+      <div class="header__phone">
+        <div class="header__number">+1 234 555-55-55</div>
+        <a href="tel:+1 234 555-55-55">
+          <img
+            src="./images/phone.png"
+            alt="tellNumber"
+            class="header__phoneLogo"
+          >
+        </a>
+      </div>
+      <h1 class="header__title">Miami<br>
+        Condo Kings
+      </h1>
+    </header>
+    <main class="pages">
+      <div class="firstPage" id="about">
```

Comment:
Better to give more meaningful names(about-us, gallery, testimonial, contact-us etc.) for your sections and also use more semantical tag `<section>` for these purposes
```suggestion
      <section class="about-us" id="about">
```